src/terms

```diff
@@ -0,0 +1,89 @@
+
+The following terms and conditions govern all use of the web3.storage website (“web3.storage” or the “Website”), the w3link/w3s.link gateway (“Gateway”) and all content, services and products available at or through the Website or Gateway (collectively, “Services”). The Services are offered subject to your acceptance without modification of all of the terms and conditions contained herein. As all data uploaded to web3.storage will be available via IPFS, the Services are also subject to the [Terms of Service of IPFS.io](https://discuss.ipfs.tech/tos) and [Privacy Policy of IPFS.io](https://discuss.ipfs.tech/privacy).
+
+If you do not agree to all the terms and conditions of this agreement (“Agreement” or “TOS”), then you may not access the Website or use any of the Services.
+
+Nitro Data LLC (“we,” “us”) may make changes to this Agreement and pricing from time to time. If you do not agree to these revisions, you may stop using the Services.
+
+#### **[Storage term](https://web3.storage/terms/#storage-term)**
+Data stored via the Services will be available via IPFS and stored on Filecoin until the expiry of the respective deals. There may be a delay between content being uploaded via the Services and being stored on Filecoin through a deal. Additionally, you may experience minor delays immediately after uploading new content to the Services while this content becomes available on the IPFS network.
+
+The maximum size of a registered upload is 750GB. Uploads greater than this size may not be registered successfully with your account, though if the data is successfully stored with web3.storage, you will still be responsible for paying for this storage.
+
+We reserve the right to terminate the Services at our sole discretion or to transfer operation of the Services to a third party or smart contract. Prior to termination or transfer, we will provide 90 days’ notice to users via email to allow users enough time to make arrangements if any (e.g. remote pinning to their own IPFS node, remote pinning to another service, or preparing to manage the renewal of their Filecoin deals on their own).
+
+All data uploaded to web3.storage is available to anyone who requests it using the correct CID. **Users should not store any private or sensitive information in an unencrypted form using web3.storage.** Deleting files from web3.storage via the site's web console or API will remove them from the file listing for a user's account, but you may be charged for at least 30 days of storage for those files. Additionally, uploaded files typically remain on our infrastructure for at minimum thirty (30) days, and nodes on the IPFS network may retain copies of the data indefinitely. **Users should not use web3.storage to store data that may need to be permanently deleted in the future.**
+
+#### **Paid Accounts**
+
+<u>Billing.  </u>
```

Comment:
why underscore billing? can use an h3?

Reply:
Just copying the formatting from the document that was approved.